new-dti-website-redesign/src/components/FeatureCard.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import IconWrapper from './IconWrapper';
+
+type FeatureCardProps = {
+  title: string;
+  body: string;
+  children: React.ReactNode;
+};
+
+export default function FeatureCard({ title, body, children }: FeatureCardProps) {
+  return (
+    <div className="p-[32px] max-w-[395px] border border-border-1 flex flex-col">
+      <IconWrapper className="icon-wrapper">{children}</IconWrapper>
+      <h5 className="text-foreground-1 pt-[16px]">{title}</h5>
```

Comment:
Nit: we don't need to explicitly set `text-foreground-1` since we're already setting it in `globals.css` [here](https://github.com/cornell-dti/idol/blob/258edcebf9682e8bc360c95d09e29978cd1ef55e/new-dti-website-redesign/src/app/globals.css#L68) in `body`, so it transmits to the `<h5>` as well.

```suggestion
      <h5 className="pt-[16px]">{title}</h5>
```